src/plotman/plot_util.py

```diff
@@ -12,11 +12,14 @@ def df_b(d):
 def get_k32_plotsize():
     return 108 * GB
 
-def human_format(num, precision):
+def human_format(num, precision, powerOfTwo=False):
     magnitude = 0
     while abs(num) >= 1000:
```

Comment:
This needs to vary as well.  Maybe start with `divisor = 1024 if power_of_two else 1000`.

Reply:
ah yes, will update

Reply:
ah yes, will update